AllReadyApp/Web-App/AllReady/Hangfire/Jobs/SendRequestStatusToGetASmokeAlarm.cs

```diff
@@ -15,14 +15,17 @@ public class SendRequestStatusToGetASmokeAlarm : ISendRequestStatusToGetASmokeAl
         public SendRequestStatusToGetASmokeAlarm(IOptions<GetASmokeAlarmApiSettings> getASmokeAlarmApiSettings, IHttpClient httpClient)
         {
             _getASmokeAlarmApiSettings = getASmokeAlarmApiSettings.Value;
+            _httpClient = httpClient;
         }
 
         public void Send(string serial, string status, bool acceptance)
         {
-            var updateRequestMessage = new { acceptance, status };
-
+            var str = JsonConvert.SerializeObject(new { acceptance, status });

```

Comment:
Regarding NewtonSoft.Json, previously we used extension method `httpClient.PostAsJsonAsync()` which will internally call the `JsonContractorResolver` to serialise the object to a json string. So we could either add that as another abstract method to the `IHttpClient` interface or use json.net.